src/libServer/LookupServer.cpp

```diff
@@ -1056,6 +1063,10 @@ Json::Value LookupServer::DSBlockListing(unsigned int page) {
 
   uint maxPages = (currBlockNum / PAGE_SIZE) + 1;
 
+  if (currBlockNum == INIT_BLOCK_NUMBER) {
```

Comment:
How about move this error checking to line 1062, just in back of the parameter assignment? This can avoid the potential misuse of this variable in the future.